code/modules/organs/internal/liver.dm

```diff
@@ -6,10 +6,10 @@
 	robotic_name = "toxin filter"
 	robotic_sprite = "liver-prosthetic"
 	toxin_type = CE_HEPATOTOXIC
-	min_bruised_damage = 25
-	min_broken_damage = 45
+	min_bruised_damage = 45
+	min_broken_damage = 65
```

Comment:
Why are you buffing these numbers? The liver is reasonably hard to hit. If anything you should be looking at organ damage dealt by inefficient filtering.